frontend/app/src/components/Header/Header.tsx

```diff
@@ -44,13 +44,18 @@ function Header({ isStale, children }: HeaderProps): ReactElement {
       // The tabindex below is required for testing.
       tabIndex={-1}
       isStale={isStale}
+      className="stAppHeader"
       data-testid="stHeader"
     >
       {showColoredLine && (
-        <StyledHeaderDecoration data-testid="stDecoration" id="stDecoration" />
+        <StyledHeaderDecoration
+          className="stAppDecoration"
```

Comment:
I left it named `stDecoration` since there are quite a few CSS hacks that hide it via the test id :( I think prefixing it with `App` would be better.. but I think I will just name everything `stDecoration`